tools/cldr-apps/js/src/index.js

```diff
@@ -3,12 +3,17 @@ import "./style.css";
 
 import { createApp } from "vue";
 import About from "./views/About.vue";
+import Waiting from "./views/Waiting.vue";
 
 function showAbout(el) {
   return createApp(About).mount(el);
 }
+function showWaiting(el) {
+  return createApp(Waiting).mount(el);
+}
 
 // The following will show up in the cldrBundle global
 export default {
   showAbout,
+  showWaiting,
 };
```

Comment:
this list -- and this file -- will be too long if we continue with this approach when everything's in the bundle; let's find a structure with separation of concerns -- it may be a lot easier if we convert all the js files into modules as in https://unicode-org.atlassian.net/browse/CLDR-14401

Reply:
does "default" do anything helpful here?

Reply:
> does "default" do anything helpful here in "export default"? maybe not needed

@btangmu yes, it's the 'default export', and is what turns into the cldrBundle object.

Reply:
> this list -- and this file -- will be too long if we continue with this approach when everything's in the bundle; let's find a structure with separation of concerns -- it may be a lot easier if we convert all the js files into modules as in https://unicode-org.atlassian.net/browse/CLDR-14401

well, 'the bundle' is intended to be the entire application. But I think it should have a mere handful of top level functions visible here.  But I will reply on your other comment: